api/client/samples/analysis_kits/stocks_to_use_price_model.py

```diff
@@ -1,239 +0,0 @@
-"""Sample API script for modeling prices based on the stocks-to-use ratio
```

Comment:
Can remove the `__init__.py` file. Not doing anything in a directory that doesn't have any `*.py` files

Reply:
done